core/conn/spj_init/src/main/resources/init_spj.sql

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
```

Comment:
Since this is a shell script, should the name really end in ".sql"? Not sure what conventions are used in the conn project, but I usually use ".sql" for scripts that are obeyed in sqlci or trafci.